exercises/practice/rail-fence-cipher/.meta/example.cpp

```diff
@@ -0,0 +1,49 @@
+#include "example.h"
```

Comment:
This line must be changed to:
```suggestion
#include "rail_fence_cipher.h"
```
During the run of the test runner, files are renamed automatically.